src/core/Form/StatusText/StatusText.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component, ReactNode } from 'react';
+import classnames from 'classnames';
+import { default as styled } from 'styled-components';
+import { baseStyles } from './StatusText.baseStyles';
+import { HtmlSpan, HtmlSpanProps } from '../../../reset';
+import { TokensProp, InternalTokensProp } from 'core/theme';
+import { withSuomifiDefaultProps } from '../../theme/utils';
+
+const baseClassName = 'fi-status-text';
+const statusTextClassNames = {
+  error: `${baseClassName}--error`,
+};
+
+export type InputStatus = 'default' | 'error' | 'success';
+
+interface InternalStatusTextProps extends HtmlSpanProps {
+  /** id */
+  id?: string;
+  /** Chip element content */
+  children?: ReactNode;
+  /** Custom class name for styling and customizing  */
+  className?: string;
+  /** Disable chip */
+  disabled?: boolean;
+  /** Status */
+  status?: InputStatus;
+}
+
+export interface StatusTextProps extends InternalStatusTextProps, TokensProp {}
+
+const StyledStatusText = styled(
+  ({
+    className,
+    children,
+    tokens,
+    status,
+    ...passProps
+  }: StatusTextProps & InternalTokensProp) => (
+    <HtmlSpan
+      {...passProps}
+      className={classnames(className, baseClassName, {
+        [statusTextClassNames.error]: status === 'error',
+      })}
+    >
+      {children}
+    </HtmlSpan>
+  ),
+)`
+  ${(tokens) => baseStyles(withSuomifiDefaultProps(tokens))}
+`;
+
+export class StatusText extends Component<StatusTextProps> {
```

Comment:
Would be good to mention these new components, StatusText and LabelText, in the PR description somewhere.
Better if those could have been introduced in separate PR, but no need to split them now.

Reply:
Agreed, but did it this way because of the work process. Didn't actually know what was needed until everything was pieced together.

Reply:
Will add to PR description.

Reply:
Added.